routes/socketio_routes.py

```
@@ -9,7 +10,10 @@ def register_socketio_handlers(registry):
     """
 
     def handle_connect():
-        """客户端连接"""
+        """客户端连接 — 未登录则拒绝，防止实时通道绕过 API 守卫。"""
```

Ruff (0.15.17)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)